e2e/utils.go

```diff
@@ -353,7 +356,9 @@ func createConfigMap(pluginPath string, c kubernetes.Interface, f *framework.Fra
 		ClusterID: fsID,
 		Monitors:  mons,
 	}}
-	conmap[0].CephFS.SubvolumeGroup = "e2e"
+	// SubvolumeGroup name is configurable and can take user specified values too.
+	// Set to "csi" to verify backward compatibility with earlier releases.
+	conmap[0].CephFS.SubvolumeGroup = "csi"
```

Comment:
this seems to be wrong, the csi will be only for older version, for newer version it should be e2e

Reply:
Oh okay, I did it as it was for only e2e tests and Subvolumegroup name was configurable.... I'll add conditions for it's modification(that it should remain `csi` for release in which subvolumegroup was not configurable). 
Is that fine?